packages/database/src/migrations/20201124054820-AddPsssSessionTable-modifies-schema.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.createTable('psss_session', {
+    columns: {
+      id: { type: 'text', primaryKey: true },
+      email: { type: 'text', notNull: true },
+      access_policy: { type: 'text', notNull: true },
+      access_token: { type: 'text', notNull: true },
+      refresh_token: { type: 'text', notNull: true },
+    },
```

Comment:
Yeah I actually thought about storing access_policy as jsonb, but because the constructor of `AccessPolicy` handles a string and parses it, there's really no need, so I leaned towards keeping this database table as dumb as possible. I can't think of instances where we would be searching by properties within the access policy, and we certainly shouldn't be manipulating it. 

There's not a huge argument either way though, so I'm happy to be talked around

Reply:
Yeah I don't think there's a particularly strong argument for it, outside of:
1. It's consistent with the existing `userSession` table (which we have already deviated from a bit here, though for good reasons imo)
2. Having `jsonb` field type will enforce that we enter data in the correct format. But given that it's created programmatically once again I don't see this as a big issue.
3. Most importantly, on the incredibly rare occasion that I want to manually inspect the policy in my db viewer, it displays in pretty json by default 😛 

So yeah, I'm not too fussed either 👍

Reply:
Well they're not super strong, but I've been swayed. Have updated in the subsequent PR :-)